### ISSUE TITLE
ImportError when pip install with Kaggle Nontebook

### ISSUE THREAD

**Issue by lisphilar** [lisphilar/covid19-sir#105]
Summary:
In Kaggle Nontebook, the following installation causes ImportError (cannot import name 'ModelBase' from covsirphy.ode.mbase)

```
!pip install covsirphy
```

CovsirPhy version 2.5.2

Environment:
Python 3.8, pipenv, WSL.

**Comment by lisphilar** [comment]
At this time, the cause is unknown.
If you find this error in Kaggle, please use the following installation code.

```
!pip install "git+https://github.com/lisphilar/covid19-sir.git#egg=covsirphy"
```

If you find the same error, please kindly notify me with this issue.
Sorry for your inconvenience.

**Comment by lisphilar** [comment]
This was fixed in version 2.5.3
